packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -67,13 +67,13 @@ curl -v -X POST \
     "algorithms": {
       "request": {
         "signature": {
-          "algorithm": "SHA-256",
+          "algorithm": "HS256",
           "scope": "REQUEST"
         }
       },
       "response": {
         "signature": {
-          "algorithm": "SHA-256",
+          "algorithm": "HS256",
           "scope": "ANY"
         }
       }
```

Comment:
I included the `response` here because was originally here. But my code did not do anything about `response`, it will be ignored.